02LessonTwo/homework/index.html

```diff
@@ -5,6 +5,34 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <h2>Julio Gonzalez III</h2>
+  <center><img style= width:400px; height:300px; src="https://scontent-dfw1-1.xx.fbcdn.net/v/t1.0-9/10410162_10153082863091789_6841706743737948299_n.jpg?oh=f9b82da217710d3c473f42eef0df02b4&oe=582F4B1B"/></center>
```

Comment:
Make sure to wrap your inline CSS style properties and values in quotes.
`<img style="property: value; property2: value2;">`